src/components/MetricChart/getDefaultDataFormatter.ts

```diff
@@ -0,0 +1,36 @@
+import {formatBytes} from '../../utils/bytesParsers';
+import {roundToPrecision} from '../../utils/dataFormatters/dataFormatters';
+import {formatToMs} from '../../utils/timeParsers';
+import {isNumeric} from '../../utils/utils';
+
+import type {ChartDataType, ChartValue} from './types';
+
+export const getDefaultDataFormatter = (dataType?: ChartDataType) => {
+    switch (dataType) {
+        case 'ms': {
+            return formatChartValueToMs;
+        }
+        case 'size': {
+            return formatChartValueToSize;
+        }
```

Comment:
I covered only current scenarios, further more data types or data types sizes (for example `'Gb' | 'Tb' | 'auto'` for `size`) may be added